src/System.IO.Ports/tests/SerialPort/BreakState.cs

```diff
@@ -12,7 +12,7 @@ namespace System.IO.Ports.Tests
     public class BreakState_Property : PortsTest
     {
         //The maximum time we will wait for the pin changed event to get firered for the break state
-        static readonly int MAX_WAIT_FOR_BREAK = 800;
+        private static readonly int s_MAX_WAIT_FOR_BREAK = 800;
```

Comment:
There's no problem making that change, but the formatter doesn't seem to then remove the s_ prefix.    Obviously if I remove the s_ prefix from the definition with a search and replace, I then have to fix up every reference manually.

Is that a bug in the formatter?